### DISCUSSION TITLE
Is anyone using a Hammerhead Karoo 3 with OsmAnd?

### DISCUSSION THREAD

**Discussion by mikehgentry** [osmandapp/OsmAnd#20222] [Q&A] [answered]
In case you have no idea what one of those is:

https://www.dcrainmaker.com/2024/05/hammerhead-new-karoo-3-review-upgrade.html

How well does it work? It's a pretty standard Android device in theory, but I'm wondering if there are any dark corners. Can you persuade it to keep the screen off between turns, for example? edit: also, does brouter work?

Thanks

**Comment by olo81** [comment, marked as answer]
I installed OsmAnd on Karoo 3. Unfortunately, the buttons don't work, only the back button does. You have to use the on-screen buttons. The screen can't be locked; I wanted to use it on a hiking trip, and after putting it in my pocket, things switch automatically. There's no access to sound, it's completely silent. After turning off the Karoo screen, it shuts down by itself after a while. OsmAnd has access to GPS and works in a way that you can sometimes use it additionally. There’s also a problem with closing OsmAnd. Karoo doesn't have an app-killing function like in normal Android, so you have to dig into the settings to do that.

**Reply by mikehgentry** [reply]
Oof, that sounds pretty rough.

I've read since posting that they still haven't released the SDK (which I suppose would smooth things, at least in theory  someone might be able to write a bridge app / plugin for sound or something)

Thanks for the reply anyway
